src/style.css

```
@@ -0,0 +1,101 @@
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 600;
+}
+
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "Avenir" , sans-serif;
+  font-weight: 300;
+  font-size: 16px;
+}
+
+.search-bar {
+  width: 100%;
+  padding: 0 8px 20px;
+}
+
+form[data-qa="big"] {
+  margin-top: 20px;
+}
+
+.search-bar-label {
+  width: 100%;
+  position: relative;
+  display: inline-block;
+}
+
+.search-bar-label::before {
+  content: "";
+  display: inline-block;
+  background: url(images/Search.svg) no-repeat center center;
+  position: absolute;
+  background-size: contain;
+}
+
+.search-bar form[data-qa="big"] .search-bar-label::before {
+  left: 26px;
+  top: 25px;
+  width: 19px;
+  height: 19px;
+}
+
+.search-bar form[data-qa="small"] .search-bar-label::before {
+  left: 13px;
+  top: 15px;
+  width: 11px;
+  height: 11px;
+}
+
+.search-bar-input {
+  width: inherit;
+  padding: 20px;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  font-family: "Avenir" , sans-serif;
+  font-weight: 300;
+}
+
+.search-bar-input:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.search-bar-input:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  outline: none;
+  font-weight: 600;
+}
+
+.search-bar-input:focus::placeholder {
+  font-weight: 300;
+}
+
+.search-bar form[data-qa="big"] .search-bar-input {
+  padding-left: 62px;
+  height: 70px;
+  line-height: 70px;
+  font-size: 16px;
+}
+
+.search-bar form[data-qa="small"] .search-bar-input {
+  padding-left: 33px;
+  height: 42px;
+  line-height: 42px;
+  font-size: 14px;
+}
```

good usage of css selectors
BUT you could just make a specific class to make it much shorter
say `search-bar-input--big` and `search-bar-input--small`
```suggestion
.search-bar-input--big {
  padding-left: 62px;
  height: 70px;
  line-height: 70px;
  font-size: 16px;
}

.search-bar-input--small {
  padding-left: 33px;
  height: 42px;
  line-height: 42px;
  font-size: 14px;
}
```